QUEST_LV_0100.tsv

```diff
@@ -8140,9 +8140,9 @@ QUEST_LV_0100_20150918_008139
 QUEST_LV_0100_20150918_008140	
 QUEST_LV_0100_20150918_008141	
 QUEST_LV_0100_20150918_008142	
-QUEST_LV_0100_20150918_008143	
+QUEST_LV_0100_20150918_008143	Complete the Pot of Guardian
```

Comment:
This isn't right. I can't come up with the right words at the moment, but the original says that the jar has been completed (it's full) because of the magic. 'Guardian' isn't in it. 

(I think this is related to the magic condensing part?)